evm/sign-message-defender.js

```diff
@@ -0,0 +1,32 @@
+const { Relayer } = require('@openzeppelin/defender-relay-client');
+const { ethers } = require('ethers');
+const { Command, Option } = require('commander');
+const { printInfo } = require('./utils');
+
+async function main(options) {
+    const { apiKey, secret, message } = options;
+
+    const relayer = new Relayer({ apiKey, apiSecret: secret });
+    const messageHex = ethers.utils.hexlify(ethers.utils.toUtf8Bytes(message));
+
+    const compactSignature = await relayer.sign({ message: messageHex });
+    const signature = ethers.utils.joinSignature(compactSignature);
```

Comment:
```suggestion
    const signature = joinSignature(compactSignature);
```
Same with `joinSignature`